docs/tools/vdb_table/data/singlestoredb.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "SingleStoreDB",
+  "links": {
+    "docs": "https://docs.singlestore.com/",
+    "github": "https://github.com/singlestore-labs",
+    "website": "https://www.singlestore.com/",
+    "vendor_discussion": "",
+    "poc_github": "",
+    "slug": "singlestoredb"
+  },
+  "oss": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "",
+    "comment": ""
+  },
+  "dev_languages": {
+    "value": [
+      "c++",
+      "go"
+    ],
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_launch_year": 2017,
```

Comment:
Wasn't vector search announced in 2024?
https://www.singlestore.com/blog/singlestore-indexed-ann-vector-search/

Reply:
Thanks for the review @svonava. We've had exact vector search since 2017: https://www.singlestore.com/blog/image-recognition-at-the-speed-of-memory-bandwidth/